src/sql/workbench/browser/designer/media/designer.css

```diff
@@ -101,4 +101,26 @@
 .designer-component .components-grid .full-row.group-header {
 	font-weight: bold;
 	line-height: 25px;
+	overflow-y: auto;
```

Comment:
is the overflow-y:auto change really needed?  what is it for?

Reply:
sorry this was a mistake